python/ray/autoscaler/aws/example-cloudwatch.yaml

```diff
@@ -27,14 +27,22 @@ auth:
 head_node:
     InstanceType: c5.4xlarge
     ImageId: latest_dlami # Deep Learning AMI (Ubuntu 18.04) Version 27.0
-    SecurityGroupIds:
-
 
 worker_nodes:
     InstanceType: r5n.2xlarge
     ImageId: latest_dlami # Deep Learning AMI (Ubuntu 18.04) Version 27.0
-    SecurityGroupIds:
 
 file_mounts: {
-    "/tmp" : "cloudwatch/prometheus.yml"
+    "~/" : "cloudwatch/prometheus.yml",
```

Comment:
Seems like this should work, but is there any reason why we don't also copy this file to `/tmp`?

Reply:
I tried to copy both file to /tmp at first but this causes a duplicate dictionary key warning and only the later file with same key actually get mounted.

Reply:
> Seems like this should work, but is there any reason why we don't also copy this file to `/tmp`?
I tried to copy both file to /tmp at first but this causes a duplicate dictionary key warning and only the later file with same key actually get mounted.

Reply:
I tried to copy both file to /tmp at first but this causes a duplicate dictionary key warning and only the later file with same key actually get mounted.

Reply:
In that case, perhaps we can provide the full path as the key:
```json
"/tmp/prometheus.yml" : "cloudwatch/prometheus.yml",
```